lang/go/go.py

```diff
@@ -0,0 +1,243 @@
+from talon import Context, Module, actions, settings
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+code.language: go
+"""
+
+# Primitive Types
+ctx.lists["self.code_type"] = {
+    "boolean": "bool",
+    "int": "int",
+    "float": "float",
+    "byte": "byte",
+    "double": "double",
+    "short": "short",
+    "long": "long",
+    "char": "char",
+    "string": "string",
+    "rune": "rune",
+    "void": "void",
+    "channel": "channel",
+}
+
+ctx.lists["user.code_keyword"] = {
+    "break": "break",
+    "continue": "continue",
+    "struct": "struct",
+    "type": "type",
+    "return": "return",
+    "package": "package",
+    "import": "import",
+    "null": "nil",
+    "nil": "nil",
+    "true": "true",
+    "false": "false",
+    "defer": "defer",
+    "go": "go",
+    "if": "if",
+    "else": "else",
+    "switch": "switch",
+    "select": "select",
+    "const": "const",
+}
+
+ctx.lists["user.code_common_function"] = {
+    # golang buildin functions
+    "append": "append",
+    "length": "len",
+    "make": "make",
+    # formatting
+    "format print": "fmt.Printf",
+    "format sprint": "fmt.Sprintf",
+    "format print L": "fmt.Println",
```

Comment:
Are you missing something here? Seems like it would make more sense to have this as "print line". Rust uses "print line".

Reply:
print line it is